change-notes/1.22/analysis-cpp.md

```diff
@@ -20,6 +20,9 @@ The following changes in version 1.22 affect C/C++ analysis in all applications.
 - The data flow library (`semmle.code.cpp.dataflow.DataFlow`) has had the
   following improvements, all of which benefit the taint tracking library
   (`semmle.code.cpp.dataflow.TaintTracking`) as well.
+  - This release includes preliminary support for flow interprocedural through
```

Comment:
'interprocedural flow' would read more naturally to me.

Reply:
Fixed. I don't know what happened there.